src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/BookingTransformer.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.health.api.model.Booking
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.BookingEntity
+import java.lang.RuntimeException
+
+@Component
+class BookingTransformer(
```

Comment:
Just wondering as there's a bit of logic here, and if it doesn't cause too much pain, is there any value in adding some unit tests for these transformers? Happy to be guided by you, but my usual way of doing things would be to add some tests if anything out of the ordinary was being done, also just for the sake of self-documentation, if nothing else.